### PR TITLE
Only redirect HTTP(s) URLs.

### PR DESCRIPTION
**Pull request by jbarrett**

### CODE REVIEW

File: lib/DDGC/Web/Controller/Root.pm [hunk 1]
```diff
@@ -113,7 +113,12 @@ sub redirect_duckco :Chained('base') :PathPart('topic') :Args(1) {
 sub redir :Chained('base') :PathPart('redir') :Args(0) {
 	my ( $self, $c ) = @_;
 	$c->stash->{not_last_url} = 1;
-	$c->session->{r_url} = $c->req->param('u');
+	my $u = URI->new( $c->req->param('u') );
+	if ( $u->scheme !~ /^http/i ) {
```
Comment (nilnilnil) [inline lib/DDGC/Web/Controller/Root.pm]: What about using if

Comment (nilnilnil) [inline lib/DDGC/Web/Controller/Root.pm]: Damn you Mobile.

Uri->secure

Comment (jbarrett) [inline lib/DDGC/Web/Controller/Root.pm]: We also use this for plain HTTP links, so it's not suitable in this instance unfortunately.

**Comment by nilnilnil** [comment]
:+1:

**Pull request merged**